### PR TITLE
Added all most possible   troubleshooting commands

### PR DESCRIPTION
**Pull request by RAJU2529**
Taken example from Windows  10 v1903 buildno 18362.207. 
added  all troubleshooting commands by using PowerShell cmdlets

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for a community contributed PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this community contributed PR. Thanks.

File: docset/windows/troubleshootingpack/invoke-troubleshootingpack.md [hunk 1]
```diff
@@ -82,6 +82,89 @@ The first command uses the **Get-TroubleshootingPack** cmdlet to get a DiagPack
 The second command invokes the troubleshooting pack stored in $Audio in unattended mode.
 The command specifies an answer file, previously created by using the **Get-TroubleshootingPack** cmdlet.
 
+### Troubleshooting commands  for all the components or devices and save all reports in C:\Diagresult
```
Comment (nenonix) [inline docset/windows/troubleshootingpack/invoke-troubleshootingpack.md]:
```suggestion
### Troubleshooting commands for all the components or devices and save all reports in C:\Diagresult
```

File: docset/windows/troubleshootingpack/invoke-troubleshootingpack.md [hunk 2]
```diff
@@ -82,6 +82,89 @@ The first command uses the **Get-TroubleshootingPack** cmdlet to get a DiagPack
...
+18. WindowsMediaPlayerMediaLibrary
+**$WindowsMediaPlayerMediaLibrary = Get-TroubleshootingPack -Path "C:\Windows\Diagnostics\System\WindowsMediaPlayerMediaLibrary"**
+**Invoke-TroubleshootingPack -Pack $WindowsMediaPlayerMediaLibrary -Result "C:\DiagResult"**
+
+19. WindowsMediaPlayerPlayDVD
+**$WindowsMediaPlayerPlayDVD = Get-TroubleshootingPack -Path "C:\Windows\Diagnostics\System\WindowsMediaPlayerPlayDVD"**
+**Invoke-TroubleshootingPack -Pack $WindowsMediaPlayerPlayDVD -Result "C:\DiagResult"**
+
+
+**Note**:: All Troubleshooting reports will be  stored under this folder  **C:\DiagResult\**
```
Comment (nenonix) [inline docset/windows/troubleshootingpack/invoke-troubleshootingpack.md]:
```suggestion
> [!NOTE]
>  All Troubleshooting reports will be  stored in the folder  **C:\DiagResult\**
```

File: docset/windows/troubleshootingpack/invoke-troubleshootingpack.md [hunk 3]
```diff
@@ -82,6 +82,89 @@ The first command uses the **Get-TroubleshootingPack** cmdlet to get a DiagPack
 The second command invokes the troubleshooting pack stored in $Audio in unattended mode.
 The command specifies an answer file, previously created by using the **Get-TroubleshootingPack** cmdlet.
 
+### Troubleshooting commands  for all the components or devices and save all reports in C:\Diagresult
+make a folder "DiagResult" in C:\
```
Comment (nenonix) [inline docset/windows/troubleshootingpack/invoke-troubleshootingpack.md]:
```suggestion
Make a folder "DiagResult" in C:\
```

File: docset/windows/troubleshootingpack/invoke-troubleshootingpack.md [hunk 4]
```diff
@@ -82,6 +82,89 @@ The first command uses the **Get-TroubleshootingPack** cmdlet to get a DiagPack
...
+then open powershell application with administrator right, type the following
```
Comment (nenonix) [inline docset/windows/troubleshootingpack/invoke-troubleshootingpack.md]:
```suggestion
Then open PowerShell with administrator permissions and use one of the following.
```

**Comment by RAJU2529** [comment]
@nenonix . Thanks sir for reviewing my additions. Thanking you for valuable work.

**Comment by get-itips** [comment]
Hello @RAJU2529 
Thanks for this contribution.
The examples must follow guidelines to be included in cmdlet's articles.
Your example should start with
Example 4: Title of example

All script examples should be formatted correctly as Powershell markdown too.

Also, I would remove the bullet numbering as they are not steps, as I understood.

Please see how other Examples in the same article are made.

Thank you for this contribution again.

**Comment by tseward** [comment]
@RAJU2529 I think the best place for these examples is in the Get-TroubleshootingPack cmdlet. Instead of listing them as examples, list them in a bullet point format in that cmdlet's `-Path` parameter.

**Comment by dariomws306** [comment]
My point of view is that these examples don't add a real value to the article. I think one of them could be added as example 4, but add all won't will look good.

**Comment by RAJU2529** [comment]
@dariomws306. before adding those commands, I have tested on my Windows machine, all are worked so I added sir.

 Everyone is having different opinion. 
let others also  comment their feedback.
Thanking you sir

**Comment by nenonix** [comment]
@RAJU2529 We really do appreciate your contributions but I agree with other comments here. Besides having technical validity (which I cannot judge but @dariomws306 and @tseward can), any content you add should follow the style and formatting of the existing content/article. All comments made here are to assist you, from various angles, and to facilitate the process of getting your changes to a point where they can be reviewed for merging, and ultimately - to get your contributions reflected in the docs.

**Comment by RAJU2529** [comment]
@nenonix . Thanking you for reply. I need your continuous cooperation for reviewing my additions in future also. 
I feel very happy, while reviewing my  contents by experts members in this regarding.

I will contribute further in other articles, if I get points.

**Comment by e0i** [comment]
@RAJU2529 

The PR is being closed as it has not met certain technical and editorial guidelines.

All Open Source contributions are welcome, and they are subject to assessment in regard to relevance, accuracy, and style. It has been to the reviewers' agreement that the PR is not suitable for the associated article.

Thank you for your time and effort. Much appreciated.

**Comment by e0i** [comment]
@officedocsbot close

**Pull request closed without merging**